src/hotspot/cpu/aarch64/interp_masm_aarch64.cpp

```diff
@@ -1783,6 +1783,8 @@ void InterpreterMacroAssembler::load_field_entry(Register cache, Register index,
   ldr(cache, Address(rcpool, ConstantPoolCache::field_entries_offset()));
   add(cache, cache, Array<ResolvedFieldEntry>::base_offset_in_bytes());
   lea(cache, Address(cache, index));
+  // Must prevent reordering of the following cp cache loads with bytecode load
```

Comment:
This is rather unclear. Where is the bytecode load to which this comment refers?

Reply:
This comment was above all the other uses of membar and was just moved here for convenience. The "bytecode load" being referred to here is `InterpreterMacroAssembler::dispatch_next`. The bytecode loading could be scheduled before the cache entry is resolved.

Reply:
The comment at the stop says "This patch adds an appropriate LoadLoad barrier after a constant pool cache field entry is loaded." But this seems to be before the constant pool cache field entry is loaded.

Reply:
I think what this wants to say is that when you've loaded the pointer to the ResolvedFieldEntry in 'cache', previous memory operations have been synched before the fields in the resolved entry are read.  I'm looking for a different word than synched.